### PR TITLE
Introduced changes in bell icon in comments form

### LINKED ISSUE #7629: The bell icon  does'nt change on clicking it.
### Please describe the problem (or idea)

> What happened just before the problem occurred? Or what problem could this idea solve?
The bell icon in comments form does'nt changes on clicking it , so it doesn't give any indication whether it's clicked or not.
Untitled_ Mar 12, 2020 3_35 PM

> What did you expect to see that you didn't?
it must be darkened when clicked.

### Please show us where to look

https://publiclab.org/ ...

### What's your PublicLab.org username?

> This can help us diagnose the issue: 

### Browser, version, and operating system

> Many bugs are related to these -- please help us track it down and reproduce what you're seeing!

****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site. 

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

### PR DESCRIPTION
**Pull request by sssash18**
Fixes #7629  (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below
**New Behaviour**
Untitled_ Mar 12, 2020 3_42 PM

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by sssash18** [comment]
@VladimirMikulic @SidharthBansal @nstjean please review the PR

**Comment by sssash18** [comment]
@VladimirMikulic I did as you suggested.Please review!

File: app/assets/javascripts/comment.js [hunk 1]
```diff
@@ -64,3 +64,8 @@ function addComment(comment, submitTo, parentID = 0) {
   }
   sendFormSubmissionAjax(data, submitTo);
 }
+
+function changeicon(messageid, buttonid){
```
Comment (VladimirMikulic) [inline app/assets/javascripts/comment.js]: The function should be camelCased and more descriptive like `changeNotificationIcon`.
`messageid` -> `messageId`
`buttonid` -> `buttonId`

**Comment by sssash18** [comment]
@VladimirMikulic I made the changes as you suggested. Thanks for guidance!

**Comment by sssash18** [comment]
@cesswairimu @SidharthBansal Kindly review this, I think it's ready for merge.

**Pull request merged**